packages/ui/src/Data/Virtualizations/VirtListView.tsx

```diff
@@ -61,16 +61,18 @@ export class VirtListView extends React.Component<IVirtsListViewProps> {
   public render() {
     return (
       <>
-        <h2>{this.props.i18nTitle}</h2>
-        <h3>{this.props.i18nDescription}</h3>
+        <div className="container-fluid">
+          <h2>{this.props.i18nTitle}</h2>
+          <h3>{this.props.i18nDescription}</h3>
+        </div>
         <ListViewToolbar {...this.props}>
           <div className="form-group">
             <OverlayTrigger
               overlay={this.getImportVirtTooltip()}
               placement="top"
             >
               <Button
-                psStyle="default"
+                psstyle="default"
```

Comment:
This should be `bsStyle` I think: https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-3/patternfly-react/src/components/Button/Button.stories.js#L25